platform-core-module/src/main/java/com/softicar/platform/core/module/license/MatchableLicense.java

```diff
@@ -19,6 +19,8 @@
 			License.APACHE_2_0,//
 			".*Apache (Software )?(License[\\s,]+)?(Version )?2\\.0.*",//
 			"apache\\.org/licenses/LICENSE-2\\.0"),
+	BOUNCY_CASTLE(License.BOUNCY_CASTLE,//
+			".*Bouncy Castle License.*"),
```

Comment:
```suggestion
			".*Bouncy Castle.*"),
```
The regex defined here must match the actual license text.
As reference, the BC license text begins with:
```
Copyright (c) 2000 - 2021 The Legion of the Bouncy Castle Inc. (https://www.bouncycastle.org) 
```